setup.py

```diff
@@ -88,7 +88,8 @@ def get_extras_require() -> Dict[str, List[str]]:
             "bokeh<2.0.0",
             "chainer>=5.0.0",
             "cma",
-            "lightgbm",
+            "lightgbm>=2.2.2",
+            "matplotlib",
```

Comment:
Is this required? If yes, could you tell me why it is necessary?

```suggestion
```

Reply:
This is required for `test_plot_importance`.